<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1154</width>
    <height>697</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>1101</width>
      <height>661</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Gráfica(Tiempo Real)</string>
     </attribute>
     <widget class="QGraphicsView" name="graphicsView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>701</width>
        <height>291</height>
       </rect>
      </property>
     </widget>
     <widget class="QCheckBox" name="Temp_box">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>120</y>
        <width>91</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Temperatura</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>40</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Presión hPa</string>
      </property>
     </widget>
     <widget class="QLabel" name="press_label">
      <property name="geometry">
       <rect>
        <x>810</x>
        <y>40</y>
        <width>41</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QCheckBox" name="Pess_box">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>150</y>
        <width>101</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Presión</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>10</y>
        <width>141</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Temperatura ºC</string>
      </property>
     </widget>
     <widget class="QLabel" name="Temp_label">
      <property name="geometry">
       <rect>
        <x>820</x>
        <y>10</y>
        <width>41</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>80</y>
        <width>63</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>ledrojo</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_4">
      <property name="geometry">
       <rect>
        <x>810</x>
        <y>80</y>
        <width>91</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>ledverde</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_5">
      <property name="geometry">
       <rect>
        <x>890</x>
        <y>80</y>
        <width>63</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>ledazul</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>880</x>
        <y>150</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Estado del Sensor:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_EstadoSensor">
      <property name="geometry">
       <rect>
        <x>990</x>
        <y>150</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QGraphicsView" name="graphicsView_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>311</y>
        <width>701</width>
        <height>281</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>340</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Humedad Relativa</string>
      </property>
     </widget>
     <widget class="QLabel" name="humidity_label">
      <property name="geometry">
       <rect>
        <x>840</x>
        <y>340</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QDial" name="dial">
      <property name="geometry">
       <rect>
        <x>1010</x>
        <y>410</y>
        <width>50</width>
        <height>64</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>820</x>
        <y>440</y>
        <width>221</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Intervalo de recepción de datos (ms):</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Base de datos</string>
     </attribute>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>30</y>
        <width>62</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Consultar</string>
      </property>
     </widget>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>701</width>
        <height>601</height>
       </rect>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_2">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>170</y>
        <width>81</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Temeratura</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_3">
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>170</y>
        <width>71</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Presión</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEdit_from">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>130</y>
        <width>110</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEdit_to">
      <property name="geometry">
       <rect>
        <x>940</x>
        <y>130</y>
        <width>110</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>110</y>
        <width>41</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Desde:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>940</x>
        <y>110</y>
        <width>41</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Hasta:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>70</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Filtrado por:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>90</y>
        <width>41</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Tiempo:</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_6">
      <property name="geometry">
       <rect>
        <x>980</x>
        <y>170</y>
        <width>71</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Humedad</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_7">
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>220</y>
        <width>71</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>ledverde</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_8">
      <property name="geometry">
       <rect>
        <x>980</x>
        <y>220</y>
        <width>71</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>ledazul</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_9">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>220</y>
        <width>81</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>ledrojo</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="widget">
     <attribute name="title">
      <string>Gráficas Humedad(Base de Datos)</string>
     </attribute>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>470</y>
        <width>41</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Desde:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>940</x>
        <y>470</y>
        <width>41</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Hasta:</string>
      </property>
     </widget>
     <widget class="QDateTimeEdit" name="dateEdit_fromHmBd_">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>490</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QDateTimeEdit" name="dateEdit_to_HmBd_">
      <property name="geometry">
       <rect>
        <x>940</x>
        <y>490</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>1011</width>
        <height>411</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout"/>
     </widget>
     <widget class="QPushButton" name="Conlut_Hm">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>480</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Consultar</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Gráficas Presión(Base de Datos)</string>
     </attribute>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>470</y>
        <width>41</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Desde:</string>
      </property>
     </widget>
     <widget class="QDateTimeEdit" name="dateEdit_from_PBd">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>490</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>1011</width>
        <height>411</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2"/>
     </widget>
     <widget class="QDateTimeEdit" name="dateEdit_to_PB">
      <property name="geometry">
       <rect>
        <x>940</x>
        <y>490</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="Conlut_P">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>480</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Consultar</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>940</x>
        <y>470</y>
        <width>41</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Hasta:</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Gráficas Temperatura(Base de Datos)</string>
     </attribute>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>1660</x>
        <y>530</y>
        <width>41</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Hasta:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Conlut_T">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>480</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Consultar</string>
      </property>
     </widget>
     <widget class="QDateTimeEdit" name="dateEdit_from_TBd">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>490</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>470</y>
        <width>41</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Desde:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>940</x>
        <y>470</y>
        <width>41</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Hasta:</string>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>1011</width>
        <height>411</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4"/>
     </widget>
     <widget class="QDateTimeEdit" name="dateEdit_to_TBd">
      <property name="geometry">
       <rect>
        <x>940</x>
        <y>490</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1154</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
